E.Deezer/Properties/AssemblyInfo.cs

```diff
@@ -1,33 +0,0 @@
-﻿using System.Resources;
```

Comment:
I'd prefer to keep this file around rather than let Visual Studio generate it for us. There's a flag you put in the csproj to disable the automatic generation.